### ISSUE TITLE
From non-pageable to pageable, the current operation will be removed when adding `operationName`

### ISSUE THREAD

**Issue by Alancere** [Azure/autorest.go#1060]
CHANGELOG:
actual:
```markdown
+	Function `*LocalRulestacksClient.ListAppIDs` has been removed
+	Function `*LocalRulestacksClient.ListCountries` has been removed
+	Function `*LocalRulestacksClient.ListPredefinedURLCategories` has been remove
```
expected:
```markdown
- Operation `*LocalRulestacksClient.ListAppIDs` has supported pagination, use `*LocalRulestacksClient.NewListAppIDsPager` instead.
- Operation `*LocalRulestacksClient.ListCountries` has supported pagination, use `*LocalRulestacksClient.NewListCountriesPager` instead.
- Operation `*LocalRulestacksClient.ListPredefinedURLCategories` has supported pagination, use `*LocalRulestacksClient.NewListPredefinedURLCategoriesPager` instead.
```

Swagger definition: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/preview/2023-09-01-preview/PaloAltoNetworks.Cloudngfw.json#L3685
```yaml
        "operationId": "LocalRulestacks_listCountries",
        "description": "List of countries for Rulestack",
        ..........................................
        "x-ms-pageable": {
          "nextLinkName": "nextLink",
          "operationName": "LocalRulestacks_listCountries"
        }
```

**Comment by tadelesh** [comment]
Explanation from Mike: https://github.com/Azure/autorest/issues/4454#issuecomment-1075164491

**Comment by tadelesh** [comment]
@Alancere please follow .NET's directive to unblock current release.
@jhendrixMSFT I believe the only thing needed for codegen is to raise an error is next operation is not correct instead of omit the generation. What do you think?

**Comment by RickWinter** [comment]
This is swagger related and stale. Please open a new issue if this is affecting TypeSpec.
